plugin/multisocket/README.md

```diff
@@ -62,6 +62,9 @@ If conducting such tests is difficult, follow these recommendations:
 
 ## Limitations
 
+The `multisocket` setting is only respected on the first zone block in the Corefile. If it is enabled there, it will
+take effect on zones. The setting is ignored in all subsequent zone blocks.
```

Comment:
I think this is directionally correct, but it might be misleading to suggest a global "first zone block". The setting is in fact per given listen address. Multisocket settings from subsequent server blocks using the same address do not apply. Consider a Corefile like this:

```
.:1053 {
  multisocket 1
}
.:1054 {
  multisocket 2
}
```

Start-up logs:

```
.:1053
.:1054
.:1054
CoreDNS-1.13.1
darwin/arm64, go1.25.1,
```



What do you think of this?

```suggestion
The `multisocket` value used for a given listen address is taken from the first server block that binds to that address in the Corefile. Subsequent server blocks using the same address will not change it. Different addresses may use different values.
```